source/common/umf_pools/disjoint_pool_config_parser.cpp

```diff
@@ -234,39 +305,39 @@ DisjointPoolAllConfigs parseDisjointPoolConfig(const std::string &config,
                  "Variable)"
               << std::endl;
 
-    std::cout << std::setw(15) << "Parameter" << std::setw(12) << "Host"
-              << std::setw(12) << "Device" << std::setw(12) << "Shared RW"
-              << std::setw(12) << "Shared RO" << std::endl;
-    std::cout
-        << std::setw(15) << "SlabMinSize" << std::setw(12)
-        << AllConfigs.Configs[DisjointPoolMemType::Host].SlabMinSize
-        << std::setw(12)
-        << AllConfigs.Configs[DisjointPoolMemType::Device].SlabMinSize
-        << std::setw(12)
-        << AllConfigs.Configs[DisjointPoolMemType::Shared].SlabMinSize
-        << std::setw(12)
-        << AllConfigs.Configs[DisjointPoolMemType::SharedReadOnly].SlabMinSize
-        << std::endl;
-    std::cout << std::setw(15) << "MaxPoolableSize" << std::setw(12)
-              << AllConfigs.Configs[DisjointPoolMemType::Host].MaxPoolableSize
-              << std::setw(12)
-              << AllConfigs.Configs[DisjointPoolMemType::Device].MaxPoolableSize
-              << std::setw(12)
-              << AllConfigs.Configs[DisjointPoolMemType::Shared].MaxPoolableSize
-              << std::setw(12)
-              << AllConfigs.Configs[DisjointPoolMemType::SharedReadOnly]
-                     .MaxPoolableSize
-              << std::endl;
-    std::cout
-        << std::setw(15) << "Capacity" << std::setw(12)
-        << AllConfigs.Configs[DisjointPoolMemType::Host].Capacity
-        << std::setw(12)
-        << AllConfigs.Configs[DisjointPoolMemType::Device].Capacity
-        << std::setw(12)
-        << AllConfigs.Configs[DisjointPoolMemType::Shared].Capacity
-        << std::setw(12)
-        << AllConfigs.Configs[DisjointPoolMemType::SharedReadOnly].Capacity
-        << std::endl;
+    // TODO: fixme, accessing config values directly is no longer allowed - API's changed
```

Comment:
I tried to do this nicely, but I'm not sure if I can... @igchor tried to make the pre-generated content in ctor, but it would have to be updated somehow in the parsing function... If this is not crucial I could update that in a sep. PR...?